docs/admin/dns.md

```diff
@@ -182,6 +182,111 @@ Address 1: 10.0.0.1
 
 If you see that, DNS is working correctly.
 
+### Troubleshooting tips.
+If above `nslookup` command does not work, here are some hints:
+
+#### Check the local DNS configuration first
+Take a look inside the resolv.conf file. (See "Inheriting DNS from the node" and "Known issues" sections for more information)
+```
+cat /etc/resolv.conf
+```
+
+You should see the search path and nameserver are set up like below: (search path may vary for different cloud providers)
+```
+search default.svc.cluster.local svc.cluster.local cluster.local google.internal c.gce_project_id.internal
+nameserver 10.0.0.10
+options ndots:5
+```
+
+#### DNS known issues
+If you are using `Alpine`(lower than 3.2) as base image, dns may not work properly because `Alpine` broke dns search paths but fixed it in some later release. Please take a look at [here](https://github.com/kubernetes/kubernetes/issues/30215) for more detail.
+
+#### Quick diagnosis
+If below appears, usually means something is wrong with kube-dns addon or Services
```

Comment:
"Errors such as the following indicate a problem with the kube-dns add-on or associated Services:"